### ISSUE TITLE
glue-vispy-viewers needs PyOpenGL

### ISSUE THREAD

**Issue by jhunkeler** [astroconda/astroconda-contrib#175]
@astrofrog FYI

```
===== testing package: glue-vispy-viewers-0.7.1-0 =====
import: 'glue_vispy_viewers'
Traceback (most recent call last):
  File "/Users/shared/iraf_conda/bldtmp/porcelain.pmVJ2MUA7O/porcelain/conda-bld/glue-vispy-viewers_1487581349474/test_tmp/run_test.py", line 27, in <module>
    import glue_vispy_viewers
  File "/Users/shared/iraf_conda/bldtmp/porcelain.pmVJ2MUA7O/porcelain/conda-bld/glue-vispy-viewers_1487581349474/_t_env/lib/python2.7/site-packages/glue_vispy_viewers/__init__.py", line 9, in <module>
    raise ImportError("The PyOpenGL package is required for this plugin")
ImportError: The PyOpenGL package is required for this plugin
```

**Comment by astrofrog** [comment]
Hmm, but pyopengl is declared as a dependency in the recipe, so it *should* work...

**Comment by pllim** [comment]
I am just installing `glueviz` with `conda` now (from `conda-forge` channel), and it appears to automatically install a bunch of other stuff including `pyopengl` 3.1. FYI.

**Comment by jhunkeler** [comment]
Fixed by #179
